Microsoft.Toolkit.Uwp.Services/Services/MicrosoftTranslator/TranslatorServiceClient.cs

```diff
@@ -0,0 +1,284 @@
+        /// </summary>
+        /// <param name="subscriptionKey">The subscription key for the Microsoft Translator Service on Azure
+        /// </param>
+        /// <remarks>
+        /// <para>You must register Microsoft Translator on https://portal.azure.com/#create/Microsoft.CognitiveServices/apitype/TextTranslation to obtain the Subscription key needed to use the service.</para>
+        /// </remarks>
+        /// <seealso cref="SubscriptionKey"/>
+        /// <seealso cref="Language"/>
+        public TranslatorServiceClient(string subscriptionKey = null)
+            : this(subscriptionKey, CultureInfo.CurrentCulture.Name.ToLower())
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TranslatorServiceClient"/> class, using the specified Subscription Key and the desired language.
+        /// </summary>
+        /// <param name="subscriptionKey">The subscription key for the Microsoft Translator Service on Azure
+        /// </param>
+        /// <param name="language">A string representing the supported language code to translate the text in. The code must be present in the list of codes returned from the method <see cref="GetLanguagesAsync"/>.</param>
+        /// <remarks>
+        /// <para>You must register Microsoft Translator on https://portal.azure.com to obtain the Subscription key needed to use the service.</para>
+        /// </remarks>
+        /// <seealso cref="SubscriptionKey"/>
+        /// <seealso cref="Language"/>
+        public TranslatorServiceClient(string subscriptionKey, string language)
+        {
+            authToken = new AzureAuthToken(subscriptionKey);
+            client = new HttpClient { BaseAddress = new Uri(_baseUrl) };
+
+            SubscriptionKey = subscriptionKey;
+            Language = language;
+        }
+
+        /// <summary>
+        /// Retrieves the languages available for speech synthesis.
+        /// </summary>
+        /// <returns>A string array containing the language codes supported for speech synthesis by <strong>Microsoft Translator Service</strong>.</returns>
+        /// <exception cref="ArgumentException">The <see cref="SubscriptionKey"/> property hasn't been set.</exception>
+        /// <remarks><para>This method performs a non-blocking request.</para>
+        /// <para>For more information, go to http://msdn.microsoft.com/en-us/library/ff512415.aspx.
+        /// </para>
+        /// </remarks>
+        public async Task<IEnumerable<string>> GetLanguagesAsync()
+        {
+            // Check if it is necessary to obtain/update access token.
+            await this.CheckUpdateTokenAsync().ConfigureAwait(false);
+
+            var content = await client.GetStringAsync(_languagesUri).ConfigureAwait(false);
+
+            XNamespace ns = "http://schemas.microsoft.com/2003/10/Serialization/Arrays";
```

Comment:
this could be a const string to be declared at the beginning of class and assigned here.